cg/cli/upload/clinical_delivery.py

```diff
@@ -81,6 +82,10 @@ def upload_clinical_delivery(context: click.Context, case_id: str, dry_run: bool
             ticket=case.latest_ticket,
         )
         trailblazer_api.add_upload_job_to_analysis(analysis_id=analysis.id, slurm_id=job_id)
+
+        slurm_upload_service: SlurmUploadService = context.obj.slurm_upload_service
+        slurm_upload_service.add_upload_job_to_analysis(slurm_id=job_id, case_id=case_id)
```

Comment:
I feel like we're doing the same thing twice here. Should we maybe remove the call to trailblazer_api

Reply:
Ah, never mind. So before we added the upload job to the RSYNC-analysis, and now we add it to both the "real" analysis and the rsync one?

Reply:
Yes